server/platform/services/upgrader/upgrader_linux.go

```diff
@@ -333,7 +333,7 @@ func extractBinary(executablePath string, filename string) error {
 			if err != nil {
 				err2 := os.Rename(tmpFileName, executablePath)
 				if err2 != nil {
-					mlog.Fatal("Unable to restore the backup of the executable file. Restore the executable file manually.")
+					mlog.Fatal("Unable to restore the backup of the executable file. RestoreContentFlaggedPost the executable file manually.")
```

Comment:
I guess this changed during a mass find and replace, can you review these again @harshilsharma63?